views/product_template.xml

```diff
@@ -8,8 +8,9 @@
         <field name="arch" type="xml">
             <group name="description" position="after">
                 <group name="website_attachments" string="Website Attachments">
-                    <field name="website_attachment_ids" nolabel="1">
-                        <tree default_order="website_name,name,id">
+                    <field name="website_attachment_ids" nolabel="1" style="width:350px">
```

Comment:
@MaganaAsiati 
Try to limitate thTry to limit the diff as much as possible to make the commits quickly understandable.

Also, pay attention to the spelling in your commit message.
And here, you're not improving the design, you're fixing the table size to make it readable.